### ISSUE TITLE
Documented the dependencies between OpenWorld and the NPC team regarding the implementation of NPCs.

### ISSUE THREAD

**Issue by navila-luna** [uchicago-cs/chiventure#679]
Will monitor the progress that the RPG-NPC team has done regarding the npc_t struct. Depending on the progress they have made in the first week of sprint 4, OpenWorld will determine whether the implementation of NPC should be done through the current mock NPCs we have or the NPC's team. This task is an attempt to be cautious of making more tasks when there's a strict deadline for sprint 4.

**Comment by navila-luna** [comment]
Over the past few days of checking in and asking, it has come to my attention that it's better to stick with the mock NPC's we've already created. The process of creating the dialogue trees isn't a task of the NPC team. Instead it's to design and implement the NPCs. What I learned through talking with the NPC-RPG team is that they rely heavily on the work of teams such as OpenWorld to be done developing rooms.
